### DISCUSSION TITLE
Announcing Mastering Pinia Course

### DISCUSSION THREAD

**Discussion by posva** [vuejs/pinia#2380] [General]
Hi Everyone 👋,

I'm really excited to finally announce my _Mastering Pinia 🍍_ course coming this Fall! 🤓

We have been working together with VueSchool to build a course that will push your knowledge like no other course!

Signup Today for an early bird discount offer 👉 [Mastering Pinia](https://masteringpinia.com/)

[![mp-image](https://github.com/vuejs/pinia/assets/664177/ce00405e-5036-4113-8258-70d2fa937572)](https://masteringpinia.com/)

**Comment by nicodevs** [comment]
This looks great! 🍍
